source/extensions/filters/network/thrift_proxy/conn_manager.cc

```diff
@@ -461,15 +462,15 @@ FilterStatus
 ConnectionManager::ActiveRpc::applyEncoderFilters(DecoderEvent state, absl::any data,
                                                   ProtocolConverterSharedPtr protocol_converter,
                                                   ActiveRpcEncoderFilter* filter) {
-  ASSERT(filter_action_ == nullptr || state == DecoderEvent::Default);
+  ASSERT(filter_action_ == nullptr || state == DecoderEvent::ContinueDecode);
   prepareFilterAction(state, data);
 
-  if (applyFilters<ActiveRpcEncoderFilter>(filter, encoder_filters_, protocol_converter) !=
-      FilterStatus::Continue) {
-    throw EnvoyException("Not supported.");
-  }
+  FilterStatus status =
+      applyFilters<ActiveRpcEncoderFilter>(filter, encoder_filters_, protocol_converter);
+  // Not support FilterStatus::StopIteration.
```

Comment:
nit: FilterStatus::StopIteration currently not supported.

Reply:
Fixed. Thanks